src/components/Sidebar/user/FeatureFlagsModal/usePersistance.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  flagInfoToFlagList,
+  Name,
+} from "@dashboard/featureFlags/availableFlags";
+import { flagListToMetadata } from "@dashboard/featureFlags/strategies/MetadataStrategy";
+import { useFlagsInfo } from "@dashboard/featureFlags/useFlagsInfo";
+import { useUserAccountUpdateMutation } from "@dashboard/graphql";
+
+export const usePersistance = () => {
```

Comment:
typo (also change filename)
```suggestion
export const usePersistence = () => {
```